uber/templates/registration/merch.html

```diff
@@ -2,6 +2,10 @@
 {% block title %}Merch Booth{% endblock %}
 {% block content %}
 
+<style type="text/css">
+    #give .merch-list { color: red; }
+</style>
```

Comment:
Minor unrelated fix; there was a ``color: red`` style applied to the entire section where the dynamic form was created, which caused even things like button text to be red, which worked but looked weird.  This fixes that so that only the part we want to be red is red.